web-marketplace/src/pages/Projects/hooks/useOffChainProjects.tsx

```diff
@@ -32,12 +32,8 @@ export const useFetchAllOffChainProjects = ({
     getAllProjectsQuery({ client: graphqlClient, enabled }),
   );
 
-  const offChainProjects = allProjectsData?.allProjects?.nodes.filter(
```

Comment:
filtering is now included in the query itself (web-marketplace/src/graphql/AllProjects.graphql) instead of manually filtering on the client side like it was the case